packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -241,13 +263,55 @@ export const CMReleasesContainer = () => {
         padding={4}
         shadow="tableShadow"
       >
-        <Flex direction="column" alignItems="stretch" gap={4}>
+        <Flex direction="column" alignItems="stretch" gap={3}>
           <Typography variant="sigma" textColor="neutral600" textTransform="uppercase">
             {formatMessage({
               id: 'content-releases.plugin.name',
               defaultMessage: 'RELEASES',
```

Comment:
Isn't the value of `content-releases.plugin.name` the string with normal casing? I'm recommending this since I think it's misleading to have the default value be different than the one in `en.json`

And I don't think it's worth having separate translation keys, as long as the difference is just the casing I feel like CSS does a good job of formatting it